ChangeLog

```diff
@@ -1,3 +1,183 @@
+  + libcrmcommon: argv argument to crm_log_preinit() is now char *const *
+  + libcrmcommon: crm_time_calculate_duration() arguments are now const
+  + libcrmcommon: deprecate CRM_ATTR_RA_VERSION
+  + libcrmcommon: deprecate crm_destroy_xml()
+  + libcrmcommon: deprecate crm_ipc_server_error
+  + libcrmcommon: deprecate crm_ipc_server_info
+  + libcrmcommon: deprecate CRM_OP_LRM_QUERY
+  + libcrmcommon: deprecate crm_str()
+  + libcrmcommon: deprecate PCMK_XE_PROMOTED_MAX_LEGACY string constant
+  + libcrmcommon: deprecate PCMK_XE_PROMOTED_NODE_MAX_LEGACY constant
+  + libcrmcommon: deprecate score2char()
+  + libcrmcommon: deprecate score2char_stack()
+  + libcrmcommon: deprecate XML_ATTR_RA_VERSION
+  + libcrmcommon: deprecate xml_get_path()
+  + libcrmcommon: deprecate XML_PARANOIA_CHECKS
+  + libcrmcommon: deprecate XML_TAG_OP_VER_ATTRS
+  + libcrmcommon: deprecate XML_TAG_OP_VER_META
+  + libcrmcommon: deprecate XML_TAG_RSC_VER_ATTRS
+  + libcrmcommon: dt argument of crm_time_get_gregorian() is now const
+  + libcrmcommon: dt argument of crm_time_get_isoweek() is now const
+  + libcrmcommon: dt argument of crm_time_get_ordinal() is now const
+  + libcrmcommon: dt argument of crm_time_get_timeofday() is now const
+  + libcrmcommon: dt argument of crm_time_get_timezone() is now const
+  + libcrmcommon: first argument to create_reply() is now const
+  + libcrmcommon: first argument to crm_copy_xml_element() is now const
+  + libcrmcommon: first argument to find_xml_node() is now const
+  + libcrmcommon: first argument to get_message_xml() is now const
+  + libcrmcommon: first argument to pcmk_ipc_name() is now const
+  + libcrmcommon: first argument to xml_patch_versions() is now const
+  + libcrmcommon: last argument to crm_write_blackbox() is now const
+  + libcrmcommon: add pcmk_rc_duplicate_id
+  + libcrmcommon: add pcmk_result_get_strings()
+  + libcrmcommon: add pcmk_result_type enum
+  + libcrmcommon: add PCMK_XE_DATE_EXPRESSION constant
+  + libcrmcommon: add PCMK_XE_OP_EXPRESSION constant
+  + libcrmcommon: add PCMK_XE_RSC_EXPRESSION constant
+  + libcrmcommon: first argument to crm_time_as_string() is now const
+  + libcrmcommon: crm_time_t argument to crm_time_log_alias() is now const
+  + libcrmcommon: argument to crm_time_get_seconds() is now const
+  + libcrmcommon: argument to crm_time_get_seconds_since_epoch() is now const
+  + libcrmcommon: sixth argument to log_data_element() is now const
+  + libcrmcommon: source argument to crm_time_set() is now const
+  + libcrmcommon: source argument to crm_time_set_timet() is now const
+  + libcrmcommon: source argument to pcmk_copy_time() is now const
+  + libpacemaker: add pcmk_check_rule()
+  + libpacemaker: add pcmk_check_rules()
+  + libpacemaker: add pcmk_show_result_code()
+  + libpacemaker: add pcmk_list_result_codes()
+  + libpacemaker: add pcmk_rc_disp_flags enum
+  + libpacemaker: pcmk_pacemakerd_status() ipc_name arg is now const
```

Comment:
This one is phrased differently from all others.  I'd expect "ipc_name argument to pcmk_pacemakerd_status() is now const".